moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/planning_component.h

```diff
@@ -171,11 +266,13 @@ class PlanningComponent
 private:
   // Core properties and instances
   rclcpp::Node::SharedPtr node_;
-  MoveItCppPtr moveit_cpp_;
+  MoveItCppPtr moveit_cpp_;  // TODO Make thread safe!
```

Comment:
I'd be interested in understanding what is currently not thread-safe.

Reply:
@sjahr where did you get the segfaults? Any stacktrace?

Reply:
The issue was likely not related to the planning pipeline. I've been calling plan() in multiple threads and each tried to write to the last_plan_solution variable. I fixed that